src/Keboola/ConfigMigrationTool/Service/OAuthService.php

```diff
@@ -30,17 +30,38 @@ public function getCredentials(string $componentId, string $id) : \stdClass
         return \GuzzleHttp\json_decode($response->getBody()->getContents());
     }
 
+    public function getCredentialsRaw(string $componentId, string $id) : \stdClass
+    {
+        $response = $this->client->get(sprintf('credentials/%s/%s/raw', $componentId, $id));
+
+        return \GuzzleHttp\json_decode($response->getBody()->getContents());
+    }
+
     public function createCredentials(string $componentId, array $account) : \stdClass
     {
+        $body = [
+            "id" => $account['id'],
+            "authorizedFor" => empty($account['email']) ? $account['owner'] : $account['email'],
+            "data" => [
+                "access_token" => $account['accessToken'],
+                "refresh_token" => $account['refreshToken'],
+            ],
+        ];
```

Comment:
proc je tahle metoda zmenena, pri te oauth migraci se vubec nepouziva, ne?

Reply:
pri migraci samotne ne, ale pouziva sa pri testoch na vytvorenie "starych" credentials.

Reply:
implementoavat neco, co se pouzije jen v testech je dost pakarna, ne? se to muze v testech vytvorit primo, anebo pres nejakej mock

Reply:
Ta funkcia uz tam aj tak bola, len som ju upravil..

Reply:
no pouziva se pro googledrivemigration a s timhle nema (mit)  nic spolecnyho

Reply:
Takze podla teba mam tuto funkciu zduplikovat v testoch aj ked uz existuje? Ta trieda je urcena pre pouzitie akoukolvek migraciou...

Reply:
Spravne by tam mal byt asi pouzity OAuth V2 klient.. Ale neviem preco je tam miesto neho tato trieda :) 
A Po pravde sa mi do toho klienta k API, ktore bude zachvilu deprekovane, moc rypat nechce.
Radsej by som to spravil takto. Pride mi to furt lepsie, nez tu funkcionalitu zprasit (a duplikovat) v testoch.

Reply:
no, ale ty ji prave nepouzivas pro migraci, ale jen zneuzivas pro test neceho jinyho, jo zduplikoval bych to, protoze je to takhle uplne divne zamotany

Reply:
Alebo zoberiem tuto triedu a presuniem ju do "test" namespacu, kedze ta migracia pre Google Drive Writer je uz "deprekovana".

Reply:
OK.